src/runtime/audience-action-type.ts

```diff
@@ -0,0 +1,19 @@
+import {InterventionType} from '../api/intervention-type';
+
+const AUDIENCE_ACTION_TYPES_VALUES = [
+  InterventionType.TYPE_BYO_CTA,
+  InterventionType.TYPE_REGISTRATION_WALL,
+  InterventionType.TYPE_NEWSLETTER_SIGNUP,
+  InterventionType.TYPE_REWARDED_AD,
+  InterventionType.TYPE_REWARDED_SURVEY,
+] as const;
+
+export type AudienceActionType = (typeof AUDIENCE_ACTION_TYPES_VALUES)[number];
+
+const values = AUDIENCE_ACTION_TYPES_VALUES as ReadonlyArray<InterventionType>;
+
+export function isAudienceActionType(
```

Comment:
Can we add some tests for this?

Reply:
IMO it would pretty much just be a test that restates the type definition, which I don't think would be very helpful. Ideally these would get tested in the users of the function.

Reply:
I think it would be helpful to show via tests that we are explicitly defining this subset of InterventionTypes to exclude Contribution and Subscription, but I'll defer this decision to you.

Reply:
Ack added test